packages/sdk/identity/src/odis/matchmaking.ts

```diff
@@ -42,6 +45,11 @@ export async function getContactMatches(
   if (sessionID) {
     body.sessionID = sessionID
   }
+  if (dekSigner) {
+    body.signedUserPhoneNumber = signWithDEK(selfPhoneNumObfuscated, dekSigner)
```

Comment:
I find the name `signedUserPhoneNumber` pretty confusing. I assumed on the first pass that this was a struct with the phone number and signature, but realize now it is only the signature. I'd recommend renaming this to be something like `hashedPhoneNumberSignature` or `phoneNumberSignature` (Although in general it is difficult to determine in this code when we are working with a hash of the phone number vs the unhashed phone number and it would be better to clear that up if possible).